lists2safebrowsing.py

```diff
@@ -480,6 +523,7 @@ def main():
                         continue
                     blocked.add(line)
 
+            output_file, log_file = get_output_and_log_files(config, section)
```

Comment:
This looks fine, but just curious - why did this move down?

Reply:
I moved it down to make it more clear when the files are used. I did this to also understand which variables were being used when to figure out how I should pull code out to create a function.